src/util/Util.js

```diff
@@ -187,6 +187,25 @@ Util.redirect = function(url, win = window, isAppLink = false) {
   }
 };
 
+Util.enrollmentRedirect = function(view) {
+  const currentViewState = view.options.appState.getCurrentViewState();
+  const ovEnrollment = window.location.href.includes('redirect_uri=https%3A%2F%2Flogin.okta.com%2Foauth%2Fcallback');
```

Comment:
Not sure if this is the best way to do things. I can't use the `URL` interface due to the following error:
`ESLint: URL is not supported in op_mini all, IE 11(compat/compat)`